<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-1EPKU6D" failures="1" tests="1" name="com.mitocode.repo.SolicitudRepoTest" time="0.202" errors="0" timestamp="2020-08-06T11:09:17 PET" skipped="0">
  <testcase classname="com.mitocode.repo.SolicitudRepoTest" name="listarSolicitudXEmpresaTest" time="0.202">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at com.mitocode.repo.SolicitudRepoTest.listarSolicitudXEmpresaTest(SolicitudRepoTest.java:30)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
]]>
    </failure>
  </testcase> <!-- listarSolicitudXEmpresaTest -->
  <system-out/>
</testsuite> <!-- com.mitocode.repo.SolicitudRepoTest -->
